setup.py

```diff
@@ -18,7 +18,7 @@
     'django-reversion>=1.8.2,<1.9',
     'django-sortedm2m',
     'django-taggit',
-    'django>=1.6,<1.8',
+    'django>=1.6,<1.8', # rq.filter: 1.6,<1.8
```

Comment:
Also, flake8 doesn't like this because PEP8 dictates two spaces between code and comment.